packages/eslint-plugin/docs/rules/prefer-nullish-coalescing.mdx

```diff
@@ -102,7 +102,7 @@ while (a ?? b) {}
 while ((a ??= b)) {}
 do {} while (a ?? b);
 for (let i = 0; a ?? b; i += 1) {}
-a ?? b ? true : false;
+(a ?? b) ? true : false;
```

Comment:
https://github.com/prettier/prettier/blob/HEAD/CHANGELOG.md#add-parentheses-for-nullish-coalescing-in-ternary-16391-by-cdignam-segment